hazelcast-hibernate53/src/main/java/com/hazelcast/hibernate/HazelcastLocalCacheRegionFactory.java

```diff
@@ -70,4 +71,14 @@ protected RegionCache createTimestampsRegionCache(final String unqualifiedRegion
 
         return new TimestampsRegionCache(this, qualifiedRegionName, instance);
     }
+
+    public long nextTimestamp() {
+         long result = instance == null ? Clock.currentTimeMillis()
+                : HazelcastTimestamper.nextTimestamp(instance);
+         return result;
+    }
+
+    public long getTimeout() {
```

Comment:
Is that necessary here?

Reply:
`nextTimestamp` override yes.
`getTimeout` is not necessary. I'll remove it.